csp-mixin/panels.libsonnet

```diff
@@ -842,5 +842,71 @@ local commonlib = import 'common-lib/common/main.libsonnet';
           },
         },
       ]),
+
+    // Azure Virtual Network
+    vn_under_ddos:
+      this.signals.azurevirtualnetwork.underDdos.asStat()
+      + g.panel.timeSeries.standardOptions.withMappings(
+        {
+          type: 'value',
+          options: {
+            '0': {
+              text: 'OK',
+              index: 0,
+            },
+            '1': {
+              text: 'Under Attack',
+              color: 'dark-red',
+              index: 1,
+            },
+          },
+        }
+      ),
```

Comment:
btw, you can add mappings to signal itself, so it is added to any panel where this signal is attached.

Reply:
Nice! I added this, thanks for pointing it out.